chapters/data-management-plan.qmd

```diff
@@ -82,22 +82,23 @@ Administrative documents, like informed consent forms and key files should be ac
 
 ## Data Assets
 
-[https://libguides.vu.nl/rdm/dmp?#s-lg-box-wrapper-15125649 > Elisa has another look at the table description, because it is quite brief]
-
-Data assets are types of data, information collected or generated in the course of research activities. It is common to provide data assets based on the data stage such as raw, processed, or analyzed. Raw Data refers to original data collected, Processed Data is data that has undergone some level of transformation or organization. Processing involves cleaning, formatting, and structuring raw data to make it more understandable and suitable for analysis. Analyzed Data usually results from statistical methods, detailed examination or interpretation.
+At the VU, we sometimes use the term 'Data Assets'. You can think of data assets as small 'parcels' of data that can change form or format throughout the research. For example, if you're sending out surveys for your research, the survey responses are considered a data asset. If, in addition to the surveys, you're also holding focus groups, the data collected from the focus group are also considered a data asset, separate from the survey results. Most projects will have more than one data asset per data stage. It is common to provide data assets based on the data stage such as raw, processed, or analysed. Raw Data refers to original data collected, Processed Data is data that has undergone some level of transformation or organisation. Processing involves cleaning, formatting, and structuring raw data to make them more understandable and suitable for analysis. Analysed Data usually results from statistical methods, detailed examination or interpretation.
 
 Here are some examples of data assets in research data management:
 
 | Data Stage     | Dataset description         | Type of data    | Format |
 |----------------|-----------------------------|-----------------|--------|
 | Raw data       | Interviews                  | Audio files     | MP3    |
 |                | Spectographic analysis      | Text files      | CSV    |
-| Processed data | Transcription of interviews | Word files      | Docx   |
+| Processed data | Transcription of interviews | Text files      | Docx   |
 |                | Data spreadsheet            | SPSS files      | SAV    |
-| Analysed data  | Regression graphic          | Photoshop files | PSD    |
+| Analysed data  | Regression graphic          | Graph           | PNG    |
 |                | Data table                  | Word file       | Docx   |
```

Comment:
```suggestion
|                | Data table                  | Text file       | Docx   |
```

I want to double check whether this may need to be changed as well? (for consistency) It may very well not be applicable!